packages/cli-kit/src/session/redirect-listener.ts

```diff
@@ -12,8 +12,9 @@ import {
 } from './post-auth.js'
 import {Abort, Bug} from '../error.js'
 import {content, info, token} from '../output.js'
-import Fastify from 'fastify'
+import {createApp, IncomingMessage, ServerResponse} from 'h3'
```

Comment:
Why are we replacing `fastify` for `h3`?

Reply:
To reduce the number of dependencies (4 vs 14). Since the HTTP servers that we need are for development, we think it's a good idea to stay lean there and not bring the sophistication of production-oriented servers like Express and Fastify.